src/Mod/Part/App/TopoShapePy.xml

```diff
@@ -115,8 +115,10 @@ importBrepFromString(str,False) to not display a progress bar.
     </Methode>
     <Methode Name="extrude" Const="true">
       <Documentation>
-        <UserDocu>Extrude the shape along a direction.
-extrude(direction, length)</UserDocu>
+        <UserDocu>Extrude the shape along a vector.
+extrude(vector) -> Shape
+--
+Shp2 = Shp1.extrude(Vector(0,0,10)) - extrude the shape 10 mm in the +Z direction.</UserDocu>

```

Comment:
shouldn't this be
```suggestion
Shp2 = Shp1.extrude(App.Vector(0,0,10)) - extrude the shape 10 mm in the +Z direction.</UserDocu>
```

Reply:
currently it's not consistent between `App.Vector` and just `Vector`. ([example](https://github.com/FreeCAD/FreeCAD/blob/a2fec5192343f862ff07702ed1cdf508a4612d0f/src/Mod/Part/App/TopoShapePy.xml#L401)).
Added a 2nd commit to homogenize this file